Content.Server/Atmos/EntitySystems/FlammableSystem.cs

```diff
@@ -285,6 +309,8 @@ public override void Update(float frameTime)
                     _alertsSystem.ClearAlert(uid, AlertType.Fire);
                     continue;
                 }
+                Logger.Debug($"IM ON FIRE!? {physics.Awake}");
+                _broadphase.RegenerateContacts(PhysicsComponent);
```

Comment:
This was an attempt to touchproxies to put the fixtures in the move buffer. It didn't appear to have any affect.